compiler/damlc/tests/src/DA/Test/DamlcUpgrades.hs

```diff
@@ -700,11 +698,6 @@ tests damlc =
           , "  - daml-stdlib"
           , "typecheck-upgrades: true"
           , "build-options:"
-          , "- --target=" <>
-                renderVersion
-                  (fromMaybe
-                    (error "DamlcUpgrades: featureMinVersion should be defined over featurePackageUpgrades")
-                    (featureMinVersion featurePackageUpgrades V2))
           ] ++ ["upgrades: '" <> path <> "'" | Just path <- pure upgradedFile]
```

Comment:
the `upgrades` field of `daml.yaml` is now unused so we can remove it too

Reply:
Is it? Aren't we using that for compiler upgrades checking?

Reply:
no, I believe the upgrades type checker uses `typecheck-upgrades`, but please confirm @dylant-da

Reply:
It uses both - we use `upgrades:` to determine the .dar, and `typecheck-upgrades:` to force typechecking.

Reply:
That said, I want to combine them

Reply:
Thanks, keeping for now then.